modules/exploits/windows/imap/mercury_login.rb

```diff
@@ -4,87 +4,82 @@
 ##
 
 class MetasploitModule < Msf::Exploit::Remote
-  Rank = AverageRanking
+  Rank = NormalRanking
 
   include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::Seh
+  include Msf::Exploit::Remote::Egghunter
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'           => 'Mercury/32 LOGIN Buffer Overflow',
-      'Description'    => %q{
+      'Name' => 'Mercury/32 4.01 IMAP LOGIN SEH Buffer Overflow',
+      'Description' => %q{
         This module exploits a stack buffer overflow in Mercury/32 <= 4.01b IMAPD
         LOGIN verb. By sending a specially crafted login command, a buffer
         is corrupted, and code execution is possible. This vulnerability was
         discovered by (mu-b at digit-labs.org).
       },
-      'Author'         => [ 'MC' ],
+      'Author'         =>
+        [
+        'MC', # Discovery and Metasploit module
+        'Ivan Racic' # Automatic targeting + egg hunter
+        ],
       'License'        => MSF_LICENSE,
       'References'     =>
         [
-          [ 'CVE', '2007-1373' ],
-          [ 'OSVDB', '33883' ],
+          ['CVE', '2007-1373'],
+          ['OSVDB', '33883']
```

Comment:
OSVDB doesn't exist any more.

```suggestion
          ['EDB', '3418']
```